index.js

```diff
@@ -144,8 +144,7 @@ class RNCallKeep {
     if (!isIOS) {
       RNCallKeepModule.setOnHold(uuid, shouldHold);
     } else {
-      // TODO: iOS hold currently not enabled on native calls
-      RNCallKeepModule.setOnHold(shouldHold)
+      RNCallKeepModule.setHeldCall(uuid, shouldHold)
```

Comment:
I'd like to keep the naming  consistent across the platforms, so I'll probably re-implement this in a slightly different way, but thanks for  pointing out the discrepancy.